File: Python/백준/Bronze/15340. Sim Card/Sim Card.py
while(True):
    a,b = map(int,input().split())
    if a==b and a==0:
        break
    aa = []
    aa.append(a*30+b*40)
    aa.append(a*35+b*30)
    aa.append(a*40+b*20)
    print(min(aa))

File: C#/백준/Bronze/15340. Sim Card/Sim Card.cs
using System;
using System.Linq;

namespace q{
    class w{
        static void Main(){
            while(true){
                int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                int a = aa[0];
                int b = aa[1];
                if (a==b && a==0){
                    break;
                }
                int[] bb = new int[3];
                bb[0] = a*40+b*20;
                bb[1] = a*35+b*30;
                bb[2] = a*30+b*40;
                Console.WriteLine(bb.Min());
            }
        }
    }
}